pkg/validation/eval.go

```diff
@@ -172,15 +185,23 @@ func (ev *Evaluator) Evaluate(rules []Rule, vm *k6tv1.VirtualMachine) *Result {
 		// we simply skip the malformed rule, the error can go unnoticed.
 		// IOW, this is a policy decision
 		if ok, err := ev.isRuleWellFormed(r, names); !ok {
-			result.Fail(r, err)
+			if r.JustWarning {
+				result.Warn(r.Message, err)
```

Comment:
I think here we should fail. Here we detect (well, try to detect) rules syntattically invalid. OTOH we want to just warning about valid rules that aren't satisfied.